test/TEST-59-EXIT-TYPE/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+TEST_DESCRIPTION="test ExitType=cgroup"
+. $TEST_BASE_DIR/test-functions
+
+do_test "$@" 59
```

Comment:
nit: the tests were reshuffled, so these should be renamed to 56 (last one is TEST-55-OOMD now)